README.md

```diff
@@ -301,8 +301,32 @@ spec:
 
 You can also achieve persistence by creating a PV in a mounted host folder.
 
-## Mounted Host Folders
-Some drivers will mount a host folder within the VM so that you can easily share files between the VM and host.  These are not configurable at the moment and different for the driver and OS you are using.  Note: Host folder sharing is not implemented in the KVM driver yet.
+## Mounting Host Folders
+`minikube mount /path/to/directory` is the recommended way to mount directories into minikube so that they can be used in your local kubernetes cluster.  The command works on all platforms.   Below is an example pod that reads from, creates and removes files from the mounted directory on the host folder `/mount-9p`.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: busybox
+spec:
+  containers:
+  - image: busybox:glibc
+    command: [ "/bin/sh", "-c", "--" ]
+    args: [ "cat /mount-9p/fromhost; echo test > /mount-9p/frompod; rm /mount-9p/fromhostremove; echo test > /mount-9p/frompodremove;" ]
+    name: busybox
+    volumeMounts:
+    - mountPath: /mount-9p
+      name: test-volume
+  volumes:
+  - name: test-volume
+    hostPath:
+      # directory location on host
+      path: /mount-9p
+
+```
+
+Additionally, some drivers are also capable of mounting a host folder within the VM so that you can easily share files between the VM and host.  These are not configurable at the moment and different for the driver and OS you are using.  This might be useful if you require a different mount solution other than the one provided by `minikube mount`.  Note: Host folder sharing is not implemented in the KVM driver yet.
```

Comment:
We should get rid of this paragraph and replace it with something along the lines of

"Some drivers themselves provide host-folder sharing options, but we plan to deprecate these in the future and they aren't configurable through minikube"...